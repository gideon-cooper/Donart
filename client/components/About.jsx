import React from 'react'

import Footer from './Footer'

export default function About () {
  const imageStyle = {
    borderRadius: '10px'
  }
  return (
    <div className="about">
      <div className="aboutPage">
        <hr />
        <h2>ABOUT</h2>
        <hr />
      </div>
      <p className="has-text-centered" style={{ margin: '6rem' }}>
                Donart is a platform for donating art towards charity. Donart empowers artists to contribute towards charities and causes they care about by using their skills rather than money. It also allows buyers to support their favourite kiwi artists and know that their money has gone to a good cause.
      </p>
      <div className="aboutPage">
        <hr />
        <h2>FOUNDERS</h2>
        <hr />
      </div>
      <div className="columns is-half" style={{ marginLeft: '17%', marginRight: '4%' }}>
        <div className="column is-two-thirds pt-6 about-flex-wrapper mx-6">
          <div className="card-flex-item mx-3">
            <img style={{ borderRadius: '10px' }} src="https://ca.slack-edge.com/T02SQPVAC-U014C2L2QE9-37da37233d7a-512" />
            <h2 className="has-text-centered">Marika Fiolitakis</h2>
          </div>
          <div className="card-flex-item mx-3">
            <img style={{ borderRadius: '10px' }} src="https://ca.slack-edge.com/T02SQPVAC-U015S11H4L8-e2ed66ddaf1a-512" />
            <h2 className="has-text-centered">Gideon Cooper</h2>
          </div>
          <div className="card-flex-item mx-3">
            <img style={{ borderRadius: '10px' }} src="https://ca.slack-edge.com/T02SQPVAC-U015SV1D742-80ebe5b6b3e0-512" />
            <h2 className="has-text-centered">Lewis Pakoti</h2>
          </div>
          <div className="card-flex-item mx-3">
            <img style={{ borderRadius: '10px' }} src="https://ca.slack-edge.com/T02SQPVAC-U0151317BDM-e8c24883182f-512" />
            <h2 className="has-text-centered">Evelyn Coulson</h2>
          </div>
        </div>
      </div>
      <Footer />
    </div>
  )
}
