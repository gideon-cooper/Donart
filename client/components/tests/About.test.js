import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer'

import About from '../About'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <About />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
