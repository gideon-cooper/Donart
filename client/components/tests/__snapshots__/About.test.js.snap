// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="about"
>
  <div
    className="aboutPage"
  >
    <hr />
    <h2>
      ABOUT
    </h2>
    <hr />
  </div>
  <p
    className="has-text-centered"
    style={
      Object {
        "margin": "6rem",
      }
    }
  >
    Donart is a platform for donating art towards charity. Donart empowers artists to contribute towards charities and causes they care about by using their skills rather than money. It also allows buyers to support their favourite kiwi artists and know that their money has gone to a good cause.
  </p>
  <div
    className="aboutPage"
  >
    <hr />
    <h2>
      FOUNDERS
    </h2>
    <hr />
  </div>
  <div
    className="columns is-half"
    style={
      Object {
        "marginLeft": "17%",
        "marginRight": "4%",
      }
    }
  >
    <div
      className="column is-two-thirds pt-6 about-flex-wrapper mx-6"
    >
      <div
        className="card-flex-item mx-3"
      >
        <img
          src="https://ca.slack-edge.com/T02SQPVAC-U014C2L2QE9-37da37233d7a-512"
          style={
            Object {
              "borderRadius": "10px",
            }
          }
        />
        <h2
          className="has-text-centered"
        >
          Marika Fiolitakis
        </h2>
      </div>
      <div
        className="card-flex-item mx-3"
      >
        <img
          src="https://ca.slack-edge.com/T02SQPVAC-U015S11H4L8-e2ed66ddaf1a-512"
          style={
            Object {
              "borderRadius": "10px",
            }
          }
        />
        <h2
          className="has-text-centered"
        >
          Gideon Cooper
        </h2>
      </div>
      <div
        className="card-flex-item mx-3"
      >
        <img
          src="https://ca.slack-edge.com/T02SQPVAC-U015SV1D742-80ebe5b6b3e0-512"
          style={
            Object {
              "borderRadius": "10px",
            }
          }
        />
        <h2
          className="has-text-centered"
        >
          Lewis Pakoti
        </h2>
      </div>
      <div
        className="card-flex-item mx-3"
      >
        <img
          src="https://ca.slack-edge.com/T02SQPVAC-U0151317BDM-e8c24883182f-512"
          style={
            Object {
              "borderRadius": "10px",
            }
          }
        />
        <h2
          className="has-text-centered"
        >
          Evelyn Coulson
        </h2>
      </div>
    </div>
  </div>
  <div
    className="footer"
  >
    <a
      href="#/About"
      onClick={[Function]}
    >
      <div
        className="firstColumn"
      >
        <h1>
          About us
        </h1>
      </div>
    </a>
    <div
      className="secondColumn"
    >
      <div>
        <h1>
          Contact us
        </h1>
        <br />
      </div>
      <div>
        <h1
          className="gideonsLineBreaks"
        >
          Copyright © 2020 – Donart® All Rights Reserved
          <br />
          Terms of Use | Privacy Policy | Disclaimer
        </h1>
      </div>
    </div>
    <div
      className="thirdColumn"
    >
      <h1>
        Legal
      </h1>
    </div>
  </div>
</div>
`;
